hack/release.toml

```diff
@@ -6,7 +6,7 @@ github_repo = "siderolabs/talos"
 match_deps = "^github.com/((talos-systems|siderolabs)/[a-zA-Z0-9-]+)$"
 
 # previous release
-previous = "v1.2.2"
+previous = "v1.2.0"
```

Comment:
Just curious - why is it v1.2.0 and not v1.2.2?

Reply:
Right now the latest is v1.2.3, as we release 1.2.4, it will be 1.2.4, and so on.